cmd/server.go

```diff
@@ -40,20 +41,40 @@ var serverCmd = &cobra.Command{
 					},
 				}
 				serverinfos = append(serverinfos, fill...)
-
 			}
 			if idFlag {
 				rowsToDisplay = len(heading)
 			}
 			render.Table(out, heading[:rowsToDisplay], serverinfos)
-
 		} else {
 			render.AsJSON(out, servers)
 		}
 		fmt.Print(out)
 	},
 }
 
+var onCmd = &cobra.Command{
+	Use:   "on",
+	Short: "Turn server on using id",
+	Run: func(cmd *cobra.Command, args []string) {
+		client.StartServer(ctx, args[0])
+	},
+}
+
+var offCmd = &cobra.Command{
+	Use:   "off",
+	Short: "Turn server off using id",
+	Run: func(cmd *cobra.Command, args []string) {
+		if forceFlag {
+			client.StopServer(ctx, args[0])
+			return
+		}
+		client.ShutdownServer(ctx, args[0])
```

Comment:
Here and other places: you should do proper error handling. Command line applications should exit with a non zero exit code on error (usually combined with a crisp/short error message that tells whats going on on stderr).

ShutdownServer as well as StopServer return errors, I think.

Reply:
I just don't expect errors haha my bad, I'll add!

Reply:
nvm, guess i do remember how to do it:
```
err := client.ShutdownServer(ctx, args[0])
		if err != nil {
			panic(err)
		}
```

Reply:
I just don't expect errors in my code haha my bad. It should be fixed by now

Reply:
Cool, thanks :slightly_smiling_face: